job-dsls/jobs/kie/main/community_buildchain_pipeline.groovy

```diff
@@ -23,7 +23,7 @@ def JENKINSFILE_REPO = 'droolsjbpm-build-bootstrap'
 def JENKINSFILE_PWD= 'kie-ci'
 def JENKINSFILE_URL = "https://github.com/${GH_ORG_UNIT}/${JENKINSFILE_REPO}"
 def JENKINSFILE_PATH = '.ci/jenkins/Jenkinsfile.release'
-def NEXUS_URL = "https://repository.jboss.org/nexus/content/groups/kie-group"
+def NEXUS_URL = "https://repository.jboss.org/nexus"
```

Comment:
is this correct? or it is used as a fragment and concatenated later into real URL ?

Reply:
it is used as a fragment and is concatenated later @NEXUS_URL/content/groups/kie-group/....